docs/api/api.yml

```diff
@@ -104,6 +100,29 @@ components:
             - OPEN_ACCESS
         license:
           type: string
+          description: The license to be used
+          enum:
```

Comment:
an enum would be nice, but it does mean another dependency outside the easy-licenses module. i would suggest to keep it as a string, and refer to the easy-licenses module in the description